wirecard/qmore.py

```diff
@@ -74,9 +74,9 @@ def init_datastorage(self, orderIdent,
         return result
 
     def init_frontend(self, amount, currency, paymentType, language,
-                      orderDescription, successUrl, cancelUrl, failureUrl,
-                      serviceUrl, confirmUrl, consumerUserAgent,
-                      consumerIpAddress, autoDeposit=None,
+                      orderDescription, customerStatement, successUrl,
```

Comment:
Also: as I understand it everything that's not stated in init_frontend will be filtered out, so we would loose the information or maybe I misunderstood this part